test/unit/Lexer-spec.js

```diff
@@ -73,8 +73,7 @@ describe('Lexer', () => {
   describe('headings', () => {
     it('depth', () => {
       expectTokens({
-        md: `
-# heading 1
+        md: `# heading 1
```

Comment:
why do we have to remove leading newlines?

Reply:
We don't actually have to. We could update the expect for the tests to expect the \n. This felt a little cleaner but don't have a strong preference.

Reply:
I would rather have the tokens updated. Just keeps the tests easier to read.